File: NetNgArch/src/ProjectBase/Desktop/SimpleApplicationStorage.cs
using System;
using System.Collections.Generic;
using System.Configuration;
using ProjectBase.BusinessDelegate;
using System.Windows.Forms;
namespace ProjectBase.Desktop
{


    public class SimpleApplicationStorage : IApplicationStorage
    {
        private readonly Dictionary<string, object> storage = new Dictionary<string, object>();


        public Object Get(string key)
        {
            Object obj;

            if (!this.storage.TryGetValue(key, out obj))
            {
                return null;
            }

            return obj;
        }


        public void Set(string key, Object obj)
        {
            this.storage[key] = obj;
        }
        public String GetAppSetting(String key)
        {
            return ConfigurationManager.AppSettings[key];
        }
        public String GetRealPath(String relativePath)
        {
            return  Application.StartupPath + relativePath;
        }
    }
}

File: JavaNgArch/src/projectbase/desktop/SimpleApplicationStorage.java
package projectbase.desktop;

import java.util.Hashtable;
import java.util.Map;

import projectbase.bd.IApplicationStorage;
import projectbase.utils.NotImplementedException;

public class SimpleApplicationStorage implements IApplicationStorage{
    private final Map<String, Object> storage = new Hashtable<String, Object>();


    public Object Get(String key)
    {
    	return storage.get(key);
    }


    public void Set(String key, Object obj)
    {
        storage.put(key, obj);
    }
    public String GetInitParameter(String key){
    	throw new NotImplementedException();
    }
    public String GetRealPath(String relativePath){
    	throw new NotImplementedException();
    }
}
